torch/ao/ns/_numeric_suite.py

```diff
@@ -522,4 +525,5 @@ def compare_model_outputs(
     float_model(*data)
     q_model(*data)
     act_compare_dict = get_matching_activations(float_model, q_model)
+    act_compare_dict = {k.replace('.stats', ''): v for k, v in act_compare_dict.items()}
```

Comment:
This leaves the NS logic to add the "stats" suffix, and removes it at the last second.  Could we achieve the same goal by removing the logic which adds the "stats" suffix in the first place? That would be cleaner.  Here is one such codepath: https://fburl.com/code/exfc8z3i

Reply:
that's good context, it's not ideal to have matching logic depend on the "stats" suffix either.  Do we have an eng estimate on what it would take to remove the dependency?

It's better than nothing to just remove stats from the API output, but it would be even cleaner to also make the internal utils work without it.